agir/carte/components/map/common.js

```diff
@@ -105,6 +105,7 @@ export function makeStyle(config, options = {}) {
         image: new Icon({
           opacity: 1,
           src: markerIcon,
+          imgSize: [40, 44],
           color: options.color ? config.color : style.primary500,
           scale: 0.75,
           anchor: [0.5, 1],
```

Comment:
Peut-être créer une variable contenant tous les attributs communs pour chaque icone ?

`const iconBasics = { opacity: 1, imgSize: [40, 44],  scale: 0.75, anchor: [0.5, 1], };`

```suggestion
        image: new Icon({
          ...iconBasics,
          src: markerIcon,
          color: options.color ? config.color : style.primary500,
       }
```